code/src/components/Form.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from "react";
+
+const Form = ({ handleFormSubmit }) => {
+  const [message, setMessage] = useState("");
+  const [alert, setAlert] = useState(false);
+  const onMessageChange = (e) => {
+    const count = message.length;
+    if (count > 140) {
+      setAlert(true);
+    } else {
+      setAlert(false);
+    }
+    setMessage(e.target.value);
+  };
+
+  const onFormSubmit = (event) => {
+    event.preventDefault();
+    if (message.length < 140) {
+      handleFormSubmit(message);
+      setMessage("");
```

Comment:
I like that it isn't possible to send when you go above 140 characters in the message, also the text becomes red. Very clear for the user.